blog/_posts/2021-12-14-sustainable-scala.md

```diff
@@ -0,0 +1,278 @@
+---
+layout: blog-detail
+post-type: blog
+by: Julien Richard-Foy, Wojciech Mazur
+title: "Sustainable Scala"
+---
+
+To what extent do programming languages affect the energy consumption of 
+programs? Where does Scala stand in that picture? Does that matter for 
+building an environmentally sustainable digital world?
+
+We found out that the most important lever is to **extend the lifetime of 
```

Comment:
```suggestion
We found out that the most important lever to reduce the energy consumption of the IT sector is to **extend the lifetime of
```

Reply:
Thank you for the suggestion. Fixed in f55c814